### PR TITLE
Release v0.3.7

### PR DESCRIPTION
**Pull request by DavidGOrtega**
- test-cml-pr (#487)
 - chore: dependancron update dependencies (#481)
 - docs: update README (#483 <- #482)
 - some README updates (#483)
 - uppercase REPO_TOKEN (#489)

### CODE REVIEW

**Comment by DavidGOrtega** [comment]
This is for internal testing of ```test-cml-pr```
I have to finish the diff report that we can also generate. Drawback is Bitbucket. Seems not to be supported.
Im still investigating.

File: src/cml.js [hunk 1]
```diff
@@ -234,6 +236,89 @@ class CML {
...
+    const branch_exists = (await exec(`git branch -r`)).includes(source);
+    if (branch_exists) {
+      const prs = await driver.prs();
+      const { url } =
+        prs.find((pr) => pr.source === source && pr.target === target) || {};
+
+      if (url) return render_pr(url);
+    } else {
+      try {
+        await exec(`git config --local user.email "david@iterative.ai"`);
```
Comment (0x2b3bfa0) [inline src/cml.js:282]: I assume that this is because we don't have a dedicated bot account yet. 🤔

Comment (DavidGOrtega) [reply src/cml.js:282]: @0x2b3bfa0 yes. Thats not finished and actually the code is going to be reiterated. We are removing exec git in favour of an isomorphic git client. 
This release is to test it in your side, checking possible scenarios that I might not have thought about.

Comment (0x2b3bfa0) [reply src/cml.js:285]: Understood, thanks!

**Pull request merged**